### PR TITLE
Add playlists/albums page, add common styles for playlist card

### PR DESCRIPTION
**Pull request by Slawisch**

### CODE REVIEW

File: backend/Perflow/Migrations/20210803162535_RunSeeding.cs [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+                table: "SongReactions",
+                columns: new[] { "Id", "SongId", "UserId" },
+                values: new object[,]
+                {
+                    { 1, 1, 1 },
+                    { 2, 3, 1 },
+                    { 3, 4, 1 }
+                });
+        }
+
```
Comment (proskurnicha) [inline backend/Perflow/Migrations/20210803162535_RunSeeding.cs]: We don't need a migration for seeding. As I know there is a script for seeding.
Ask @krolebord more about it.

File: backend/Perflow/Migrations/20210805133743_AlbumReactionSeed.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                table: "Albums",
+                newName: "isSingle");
+
+            migrationBuilder.RenameColumn(
+                name: "IsPublished",
+                table: "Albums",
+                newName: "isPublished");
+        }
+    }
+}
```
Comment (proskurnicha) [inline backend/Perflow/Migrations/20210805133743_AlbumReactionSeed.cs]: We definitely don't need 3 different files with migrations.

File: backend/Perflow/Services/Implementations/AlbumReactionService.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+                await context.AlbumReactions.AddAsync(albumReaction);
+
+                await context.SaveChangesAsync();
+            }
+            else
+            {
+                throw new ArgumentException("Reaction already exists");
+            }
+        }
```
Comment (proskurnicha) [inline backend/Perflow/Services/Implementations/AlbumReactionService.cs:55]: Use Return Early Pattern
https://medium.com/swlh/return-early-pattern-3d18a41bba8

File: backend/Perflow/Services/Implementations/AlbumReactionService.cs [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+            {
+                context.AlbumReactions.RemoveRange(reactions);
+
+                await context.SaveChangesAsync();
+            }
+            else
+            {
+                throw new ArgumentException("Reaction does not exist");
+            }
+        }
```
Comment (proskurnicha) [inline backend/Perflow/Services/Implementations/AlbumReactionService.cs:74]: And here

File: backend/Perflow/Services/Implementations/ArtistReactionService.cs [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                };
+
+                await context.ArtistReactions.AddAsync(artistReaction);
+
+                await context.SaveChangesAsync();
+            }
+            else
+            {
+                throw new ArgumentException("Reaction already exists");
+            }
```
Comment (proskurnicha) [inline backend/Perflow/Services/Implementations/ArtistReactionService.cs:39]: here and in the next places

File: frontend/src/app/services/reaction.service.ts [hunk 6]
```diff
@@ -26,4 +28,46 @@ export class ReactionService {
...
+    );
+    return response;
+  }
+
+  removeAlbumReaction(albumId: number, userId: number) {
+    const response = this.httpService.postClearRequest(
+      '/api/AlbumReaction/removeLike',
+      { albumId, userId }
+    );
+    return response;
```
Comment (proskurnicha) [inline frontend/src/app/services/reaction.service.ts]: I would say that you don't need to assign results to const, you could just return. Here and above in methods

**Review by proskurnicha — CHANGES_REQUESTED** [review]

File: backend/Perflow/Common/DTO/Users/UserReadDTO.cs [hunk 7]
```diff
@@ -11,6 +11,11 @@ public class UserReadDTO
         public string Email { get; set; }
         public DateTimeOffset Birthday { get; set; }
         public bool Gender { get; set; }
+        public string Description { get; set; }
+        public string Country { get; set; }
+        public string Password { get; set; }
+        public string Salt { get; set; }
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/Common/DTO/Users/UserReadDTO.cs]: Remove Password and Salt from dto

File: backend/Perflow/DataAccess/Context/ModelBuilderExtensions.cs [hunk 8]
```diff
@@ -25,6 +26,7 @@ public static void Configure(this ModelBuilder modelBuilder)
             modelBuilder.ApplyConfiguration(new UserSettingsConfiguration());
             modelBuilder.ApplyConfiguration(new NotificationConfiguration());
             modelBuilder.ApplyConfiguration(new ArtistFollowerConfiguration());
+            Seed(modelBuilder);
```
Comment (NikitaKrasnovSW) [inline backend/Perflow/DataAccess/Context/ModelBuilderExtensions.cs]: Delete seeding and seeding migrations

**Review by NikitaKrasnovSW — CHANGES_REQUESTED** [review]
Update your previous pull request and then this one and most of the comments will be gone. Next time try to avoid merging branches in a new one because it's different to review what you really changed in the pull request since it contains a lot of the code from different branch

**Comment by Slawisch** [comment]
I've fixed all of problems from previous pull request and this one only in this branch. So, all changes exist in this branch. Previous ('artist-list') I can delete.

**Review by proskurnicha — APPROVED** [review]

**Pull request merged**